tests/deployment_test.go

```diff
@@ -217,23 +223,21 @@ func (s *DeploymentSuite) TestDescribeDeployment_RegisterTaskQueue_ConcurrentPol
 
 		resp, err := s.FrontendClient().DescribeDeployment(ctx, &workflowservice.DescribeDeploymentRequest{
 			Namespace:  s.Namespace(),
-			Deployment: workerDeployment,
+			Deployment: d,
 		})
 		if !a.NoError(err) {
 			return
 		}
 		a.NotNil(resp.GetDeploymentInfo())
 		a.NotNil(resp.GetDeploymentInfo().GetDeployment())
 
-		a.Equal(seriesName, resp.GetDeploymentInfo().GetDeployment().GetSeriesName())
-		a.Equal(buildID, resp.GetDeploymentInfo().GetDeployment().GetBuildId())
+		a.True(d.Equal(resp.GetDeploymentInfo().GetDeployment()))
 
-		if !a.Equal(numberOfDeployments, len(resp.GetDeploymentInfo().GetTaskQueueInfos())) {
+		if len(resp.GetDeploymentInfo().GetTaskQueueInfos()) < 1 {
```

Comment:
Will this allow the `Eventually` to complete without waiting for `len(resp.GetDeploymentInfo().GetTaskQueueInfos()) >= 1` to be true? So it could complete without ever checking the following assertions? I think you would want to keep this `if len < 1 { return }`, but put an `a.GreaterThan(0, len(list))` above it so that the `Eventually` keeps retrying until that condition is satisfied and it can move on to the other checks

Reply:
unless the goal is to make this work when there are zero task queues expected..

Reply:
good catch! fixed it.